lib/builders.js

```diff
@@ -1,25 +1,25 @@
-const fs = require('fs');
-const chalk = require('chalk');
-const request = require('request-promise');
-_ = require('lodash');
+const fs = require("fs");
```

Comment:
We typically use single quotes. It looks like something converted all of them in this file?